### ISSUE TITLE
Fabric Dynmap - Block changes are registered but not automatically rendered

### ISSUE THREAD

**Issue by RudiKleinert** [webbukkit/dynmap#3849]
**Issue Description:** During a project drying out an ocean monument using temporary separator walls made of sand and sponges I noticed that the map was not showing the current state of the world. I studied it together with a member of the discord support team and it seems that block changes arenot being consistently picked up or at least don't find their way into the map. The /stats and /triggerstats show all block changes as I would expect them. But the respective changes don't get rendered into the map.

* **Dynmap Version:** Dynmap version: core=3.4-828, plugin=3.4
* **Server Version:** Minecraft 1.19 with Fabric Loader 0.14.9
* **Pastebin of Configuration.txt:** https://pastebin.com/3ifaBwVw
* **Server Host (if applicable):** Selfhosted
* **Pastebin of crashlogs or other relevant logs:** N/A
* **Other Relevant Data/Screenshots:** The issue was intensely discussed on the support discord beforehand. See, https://discord.com/channels/722722769950998560/729067379405095005/1017066446536323113 The issue was also reproduced by a support member, see https://discord.com/channels/722722769950998560/729067379405095005/1017341717206286366
* **Steps to Replicate:**  Breaking large walls of sand seems to do the trick. But also other changes seem to not be picked up correctly. Some changes are being rendered instantly while others never get rendered until I do a manuel radiusrender. It feels like sometimes the queue gets "clogged up".

 [x ] *I have looked at all other issues and this is not a duplicate*  
 [x ] *I have been able to replicate this*

**Comment by RudiKleinert** [comment]
I found that my map doesn't render most of the changes. It doesn't seem to be connected to sand or water but most of the changes are not automatically updated.

**Comment by JurgenKuyper** [comment]
were you perhaps using a proxy to see the map on your domain? could be the updates were being rendered fine, but the proxy takes time to see the changes, does it work on the direct ip?

**Comment by RudiKleinert** [comment]
Nope, the Server runs on a (physical) Linux machine in my house and is accessible online. I access it through my domain via the direct ip and port. So no proxy in between.

**Comment by JurgenKuyper** [comment]
Do you have proxying enabled on your domain?
